services/base.spec.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const assert = require('assert');
+const { expect } = require('chai');
 const sinon = require('sinon');
 
```

Comment:
Adding `require('../lib/register-chai-plugins.spec');` here allows to run tests only from this file (using command line or IDE). Otherwise we will get `Error: Invalid Chai property: calledOnce ...`